interface/patient_file/summary/add_edit_issue.php

```diff
@@ -840,5 +841,22 @@ function divclick(cb, divid) {
  Calendar.setup({inputField:"form_end", ifFormat:"%Y-%m-%d", button:"img_end"});
  Calendar.setup({inputField:"form_return", ifFormat:"%Y-%m-%d", button:"img_return"});
 </script>
+
+<!--Page Form Validations-->
+<?php require_once($GLOBALS['srcdir']."/validation/validation_script.js.php");?>
+<?php $collectThis=collectValidationPageRules("add_edit_issue",$_SERVER['PHP_SELF'])?>
```

Comment:
I see what you are saying. In that case, then there would be no need for validate_id then, if the goal is to completely automate it. But note you still can't automate the form.id, thus are always stuck with setting things up in each script (let me know if I am incorrect here). Also with just using the path/filename would lose out on following situations:
1. Using same validation set for different scripts
2. Supporting more than 1 form in a script

Sounds like a development design decision needs to be made :)